### DISCUSSION TITLE
Reset simulation automatically

### DISCUSSION THREAD

**Discussion by FoFFolo** [mesa/mesa#2064] [Q&A] [answered]
Hello, I didn't manage to find a way to reset the simulation automatically. There exists some way to do so or is it not intended?

**Comment by EwoutH** [comment]
Could you just initialize a new model instance with the same seed?

**Reply by rht** [reply, marked as answer]
That wouldn't necessarily be automatic. I think @FoFFolo wants to have an equivalent of having the "reset" button automatically "pressed" for a certain condition. I haven't replied in time because the mesa-viz-tornado is not actively developed anymore, and it's somewhat tiring to have to say this every time.

**Reply by FoFFolo** [reply]
Ok now I understood, thank you so much.
